<?xml version="1.0" ?>
<testsuite name="TestRoutes.TestUsersBlueprint-20240411100952" tests="5" time="0.016" failures="0" errors="1">
	<testcase classname="TestRoutes.TestUsersBlueprint" name="test_login" time="0.009"/>
	<testcase classname="TestRoutes.TestUsersBlueprint" name="test_login_fail" time="0.003"/>
	<testcase classname="TestRoutes.TestUsersBlueprint" name="test_prediction" time="0.001"/>
	<testcase classname="TestRoutes.TestUsersBlueprint" name="test_save" time="0.002"/>
	<testcase classname="TestRoutes.TestUsersBlueprint" name="test_register" time="0.002">
		<error type="TypeError" message="TestUsersBlueprint.test_register() takes 1 positional argument but 2 were given"><![CDATA[Traceback (most recent call last):
  File "C:\Program Files\WindowsApps\PythonSoftwareFoundation.Python.3.11_3.11.2544.0_x64__qbz5n2kfra8p0\Lib\unittest\mock.py", line 1378, in patched
    return func(*newargs, **newkeywargs)
           ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
TypeError: TestUsersBlueprint.test_register() takes 1 positional argument but 2 were given
]]></error>
	</testcase>
	<system-out><![CDATA[]]></system-out>
	<system-err><![CDATA[]]></system-err>
</testsuite>
